src/createRollupConfig.ts

```diff
@@ -125,22 +126,16 @@ export function createRollupConfig(opts: TsdxOptions) {
           };
         },
       },
-      typescript({
-        typescript: require('typescript'),
-        cacheRoot: `./.rts2_cache_${opts.format}`,
-        tsconfig: opts.tsconfig,
-        tsconfigDefaults: {
-          compilerOptions: {
-            sourceMap: true,
-            declaration: true,
-            jsx: 'react',
-          },
-        },
-        tsconfigOverride: {
-          compilerOptions: {
-            target: 'esnext',
-          },
-        },
+      ts({
+        tsconfig: tsconfig => ({
```

Comment:
Best bet would be to check if `opts.tsconfig` is given, if not pass path to the TSDX default tsconfig.json. Something like mine

```json
{
  "$schema": "http://json.schemastore.org/tsconfig",
  "compilerOptions": {
    "skipLibCheck": true,
    "alwaysStrict": true,
    "newLine": "LF",
    "declaration": true,
    "declarationMap": true,
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": false,
    "allowSyntheticDefaultImports": true,
    "forceConsistentCasingInFileNames": true,
    "importHelpers": true,
    "incremental": false,
    "jsx": "react",
    "lib": ["dom", "esnext"],
    "module": "esnext",
    "moduleResolution": "node",
    "noEmit": true,
    "strict": true,
    "sourceMap": true,
    "stripInternal": true,
    "noUnusedLocals": true,
    "noImplicitReturns": true,
    "resolveJsonModule": true,
    "noUnusedParameters": true,
    "target": "esnext"
  }
}
```

Some of them will be automatically ignored anyway.

Also would be good to provide some options for "export tsconfig" and "export babel config" to the root of the project that is using TSDX.